caom2utils/caom2utils/fits2caom2.py

```diff
@@ -1248,7 +1306,9 @@ def _get(self, caom2_element, extension=0):
         """
         ObsBlueprint.check_caom2_element(caom2_element)
         if extension:
-            ObsBlueprint.check_chunk(caom2_element)
+            # TODO - figure out what the implications of removing the
```

Comment:
The _get instance of check_chunk has been removed. There is a lot of code that specifies an extension of 0 instead of None, and with the _get method, neither the content of the blueprint or the resulting FITS file content are affected, so I changed the one location instead of the many.